docs/loading/RoutineLoad.md

```diff
@@ -63,7 +63,7 @@ Routine Load now supports consuming CSV and JSON format data from a Kafka cluste
 
 ## Create a Routine Load job
 
-The following two examples describe how to consume CSV-format and JSON-format data in Kafka, and load the data into StarRocks by creating a Routine Load job. For detailed instruction and reference, see [CREATE ROUTINE LOAD](../sql-reference/sql-statements/data-manipulation/CREATE%20ROUTINE%20LOAD.md).
+The following three examples describe how to consume CSV-format, JSON-format and Avro-format data in Kafka, and load the data into StarRocks by creating a Routine Load job. For detailed instruction and reference, see [CREATE ROUTINE LOAD](../sql-reference/sql-statements/data-manipulation/CREATE%20ROUTINE%20LOAD.md).
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/19f61f52-a1ca-4288-93b9-54d3be8ba2dc)
-> For detailed syntax and parameter descriptions?